### DISCUSSION TITLE
Provide support for PHP

### DISCUSSION THREAD

**Discussion by msankhala** [runsidekick/sidekick#53] [Ideas]
I really like the capabilities of sidekick to allow us to put tracepoint and log point in production applications without requiring redeployment. The only thing I miss is that sidekick isn't available for PHP.

>PHP is the most used server-side programming language on the web. In fact, 79.2% of all websites rely on PHP to some degree, making it one of the most popular languages among programmers and web developers due to its widespread use.

Source: https://kinsta.com/php-market-share/

PHP can run in different mods.

1. Cli mode
2. On servers
    1. Apache + mod_php
    2. Apache + php-fpm
    3. nginx + php-frpm

Few references for comparison:
https://stackoverflow.com/questions/29403998/is-nginx-php-fpm-is-suppose-be-much-faster-than-apache-mod-php
https://serverfault.com/questions/174008/comparing-nginxphp-fpm-to-apache-mod-php

I would really like to see sidekick being available for PHP both in CLI mode and in Server mode.

**Comment by boroskoyo** [comment]
We can use this conversation to observe the demand for a PHP debugger. Any cooperation will be highly appreciated for such a contribution
